support/cas-server-support-oauth/src/main/java/org/apereo/cas/client/CasBackchannelTicketValidator.java

```diff
@@ -0,0 +1,30 @@
+package org.apereo.cas.client;
+
+import org.apereo.cas.CentralAuthenticationService;
+import org.jasig.cas.client.authentication.AttributePrincipalImpl;
+import org.jasig.cas.client.validation.Assertion;
+import org.jasig.cas.client.validation.AssertionImpl;
+import org.jasig.cas.client.validation.TicketValidator;
+
+/**
+ * This is a ticket validator for pac4j client that uses CAS back channels to validate ST.
+ *
+ * @author Kirill Gagarski
+ * @since 6.1.0
+ */
+public class CasBackchannelTicketValidator implements TicketValidator {
```

Comment:
May be the name of the class to be more clearer something like `CasServerApiBasedTicketValidator` ?